apps/mobile/src/urql-shared/updates/index.js

```diff
@@ -71,5 +71,32 @@ export default {
         cache.invalidate(cache.keyOfEntity({ __typename: 'Post', id: args.id }), 'myEventResponse')
       }
     }
+  },
+  Subscription: {
+    comments: (result, args, cache, info) => {
+      console.log('!!!!! comments - result, args, info:', result, args, info)
+    },
+    updates: (result, args, cache, info) => {
+      const update = result?.updates
+
+      switch (update?.__typename) {
+        case 'Message': {
+          console.log('!!!! updates TODO: new Message. Increment Messages tab badge', result, args)
```

Comment:
??

Reply:
As mentioned in the perhaps tl;dr description:), managing badges is out of scope in this PR and "coming soon". The logging, and the 999 badge on the Messages tab are meant as book marks.